tools/Az/Az.psd1

```diff
@@ -297,7 +297,6 @@ Az.PrivateDns
 
 Az.RecoveryServices
 * Update references in .psd1 to use relative path
-* Added SoftDelete feature for VM and added tests for softdelete
```

Comment:
Why is this line removed

Reply:
this message was incorrectly been added , already existed in version 2.0.1